1.10/README.md

```diff
@@ -1,17 +1,44 @@
-Nginx 1.10 server and a reverse proxy server docker image
+Nginx 1.10 server and a reverse proxy server Docker image
 =========================================================
 
-The `centos/rh-nginx-110-centos7` image provides an nginx 1.10 server and a reverse proxy server. The image can be used as a base image for other applications based on nginx 1.10 web server.
+This container image includes Nginx 1.10 server and a reverse server for OpenShift and general usage.
+Users can choose RHEL based image.
+The RHEL image is available in the [Red Hat Container Catalog](https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/nginx-110-rhel7)
+as registry.access.redhat.com/rhscl/nginx-110-rhel7.
 
 
-To pull the `centos/rh-nginx-110-centos7` image, run the following command as root:
+Description
+-----------
+
+Nginx is a web server and a reverse proxy server for HTTP, SMTP, POP3 and IMAP 
+protocols, with a strong focus on high concurrency, performance and low memory usage. The container 
+image provides a containerized packaging of the nginx 1.10 daemon. The image can be used 
+as a base image for other applications based on nginx 1.10 web server. 
+Nginx server image can be extended using source-to-image tool.
+
+
+Usage
+-----
+
+To build a simple [sample-app](https://github.com/sclorg/nginx-container/tree/master/1.10/test/test-app) application
+using standalone [S2I](https://github.com/openshift/source-to-image) and then run the
+resulting image with [Docker](http://docker.io) execute:
+
+*  **For RHEL based image**
+    ```
+    $ s2i build https://github.com/sclorg/nginx-container.git --context-dir=1.10/test/test-app/ rhscl/nginx-110-rhel7 nginx-sample-app
+    $ docker run -p 8080:8080 nginx-sample-app
+    ```
```

Comment:
There is `centos/nginx-110-centos7` images as well https://hub.docker.com/r/centos/nginx-110-centos7/, please add it there same as in case of 1.8.